### PR TITLE
Create Pcgamer.com.xml

### PR DESCRIPTION
**Pull request by signingkeys**
fixes https://github.com/EFForg/https-everywhere/pull/9743

### CODE REVIEW

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!--
+	Nonfunctional hosts in *.pcgamer.com:
+	
+	r: connection refused
+	    asus.pcgamer.com (Server not found)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: This host does not exist at all, you can just remove it.

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	    dl.pcgamer.com (Server not found)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: I got a timeout.

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	    promo.pcgamer.com (Server not found)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: Does not exist

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	    www.sl.pcgamer.com (Server not found)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: Does not exist

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	
+	HTTPS connection refused:
+       	    email.pcgamer.com (HTTPS refused)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: I got a mismatch

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+	    media.pcgamer.com (Invalid Hostname)
+	    sl.pcgamer.com (HTTPS refused)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: I got an expired cert

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	
+	r: connection refused
+	    asus.pcgamer.com (Server not found)
+	    dl.pcgamer.com (Server not found)
+	    promo.pcgamer.com (Server not found)
+	    www.sl.pcgamer.com (Server not found)
+	
+	HTTPS connection refused:
+       	    email.pcgamer.com (HTTPS refused)
+	    media.pcgamer.com (Invalid Hostname)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: I got a timeout

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	    sl.pcgamer.com (HTTPS refused)
+	    weekender.pcgamer.com (HTTPS refused)
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: I got a mismatch

File: src/chrome/content/rules/Pcgamer.com.xml [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+-->
+<ruleset name="Pcgamer" platform="mixedcontent">
+
+	<target host="pcgamer.com" />
+	<target host="www.pcgamer.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Pcgamer.com.xml]: See also `club.pcgamer.com`

**Comment by signingkeys** [comment]
@J0WI Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**